plugins/modules/luks_device.py

```diff
@@ -217,6 +217,47 @@
             - "Will only be used on container creation."
         type: int
         version_added: '1.5.0'
+    perf_same_cpu_crypt:
+        description:
+            - "This option allows user to perform encryption using the same cpu that IO was submitted on."
+            - "The default is to use an unbound workqueue so that encryption work is automatically balanced between available CPUs."
+            - "This option is only relevant for open action."
+        type: bool
+        default: false
+        version_added: '2.3.0'
+    perf_submit_from_crypt_cpus:
+        description:
+            - "This option allows user to disable offloading writes to a separate thread after encryption."
+            - "There are some situations where offloading write bios from the encryption threads to a single thread degrades performance sig‐
+              nificantly."
```

Comment:
What is a `(write) bios`? Do you mean `write operations`? I think that needs to be explained, and/or correctly capitalized. (Same in the next paragraph.)
```suggestion
            - "There are some situations where offloading write bios from the encryption threads to a single thread degrades performance significantly."
```

Reply:
This is copied directly from the `man cryptsetup` . I don't know what author means by saying `(write) bios`.
Will fix typos.

Reply:
I'm not sure whether options should be added where we don't really understand what they do. I would guess this is about Block I/O (https://lwn.net/Articles/26404/, https://stackoverflow.com/questions/14574052/the-bio-structure-in-the-linux-kernel) somehow.